sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -609,178 +701,16 @@ def __init__(self, **kwargs):
 
     @classmethod
     def _from_generated(cls, err):
-        return [cls(code=error.code, message=error.message) for error in err] if err else None
+        return [cls(code=error.code, message=error.message) for error in err] if err else []
```

Comment:
out of curiosity, why the difference in returning `[]` instead of `None`?

Reply:
I tried to match what the service does here. they seem to send a list of errors, even if there are none (so empty list)

Reply:
interesting. Thank you!